packages/connectors-web/src/utils/resolveConnector.ts

```diff
@@ -1,5 +1,6 @@
 import type { Config, Connector } from "wagmi";
 import type { AuthClient, AuthSession } from "@alchemy/auth";
+import { getConnectors } from "@wagmi/core";
```

Comment:
did not know this was a thing, thanks!

Reply:
np, i just learned about it this week. looks like all it does is memoize the connectors, which could lead to a perf improvement for consumers.